packages/shared/src/hooks/mutationSubscription/useMutationSubscription.ts

```diff
@@ -15,7 +15,7 @@ export const useMutationSubscription = ({
   matcherRef.current = matcher;
 
   useEffect(() => {
-    const subscriptionId = (mutationSuccessSubscribers.size + 1).toString();
+    const subscriptionId = crypto.randomUUID();
```

Comment:
If we need a stable id that do not change overtime maybe `useId` can be a better alternative?

Reply:
that one changes based on the component tree no? So it would be less stable

Reply:
used `uuid` package, we had it from before and compatibility wise its better